### PR TITLE
[MONIT-28280] Add new metrics related with the Memory buffer

### PR DESCRIPTION
**Pull request by laullon**

### CODE REVIEW

File: proxy/src/main/java/com/wavefront/agent/handlers/AbstractSenderTask.java [hunk 1]
```diff
@@ -55,6 +55,7 @@
   final Counter blockedCounter;
   final Counter bufferFlushCounter;
   final Counter bufferCompletedFlushCounter;
+  private final Histogram metricSize;
```
Comment (amitw-vmware) [inline proxy/src/main/java/com/wavefront/agent/handlers/AbstractSenderTask.java:58]: [future] we may want to call it atomSizes (or metricSizes in the future) but we aren't consistent about the term atom or metric so this works :)

**Review by amitw-vmware — APPROVED** [review]
LGTM

File: proxy/src/main/java/com/wavefront/agent/handlers/AbstractSenderTask.java (hunk 1 above)
Comment (laullon) [reply proxy/src/main/java/com/wavefront/agent/handlers/AbstractSenderTask.java:58]: sure

**Review by joannak-vmware — APPROVED** [review]
LGTM

**Pull request merged**